frontends/ol-components/src/components/ThemeProvider/buttons.ts

```diff
@@ -1,25 +1,18 @@
 import type { ThemeOptions } from "@mui/material/styles"
-import { pxToRem } from "./typography"
 
-const buttonComponent: NonNullable<ThemeOptions["components"]>["MuiButton"] = {
+const buttonBaseComponent: NonNullable<
+  ThemeOptions["components"]
+>["MuiButtonBase"] = {
   defaultProps: {
-    size: "large",
-    disableElevation: true,
+    disableRipple: true,
```

Comment:
Although I decided not to use MUI's button component (see above) I did add customization for their `ButtonBase`, specifically disabling the ripple and using browser's default `:focus-visible` behavior.

MUI's ButtonBase is used by several of their other components, e.g., Tabs, Pagination, all use ButtonBase.